src/main/webapp/javascript/login.js

```diff
@@ -0,0 +1,116 @@
+function collectDto(){
+    var usernameTxt = document.getElementById("username-txt");
+    var password1Txt = document.getElementById("password1-txt");
+    var password2Txt = document.getElementById("password2-txt");
+    var name = document.getElementById("name-txt");
+    var surname = document.getElementById("surname-txt");
+    var phoneNumber = document.getElementById("phoneNumber-txt");
+    var pwd1 = password1Txt.value;
+    var pwd2 = password2Txt.value;
+    if (pwd1 !== pwd2) {
+        showError("Passwords doesn't match!")
+        return;
+    }
+    var dto = {
+        "username": usernameTxt.value,
+        "password": pwd1,
+        "name": name.value,
+        "surname": surname.value,
+        "phoneNumber": phoneNumber.value
+    }
+    register(dto)
+}
+function switchRegistration() {
+    hideError();
+    var checkbox = document.getElementById("register-cb");
+    var checkbox_carRegButton = document.getElementById("register_car-grp");
+    var checkbox_carReg = document.getElementById("register_car-cb");
+    var pwd2 = document.getElementById("password2-grp");
+    var loginBtn = document.getElementById("login-btn");
+    var registerBtn = document.getElementById("register-btn");
+    var name = document.getElementById("name-grp");
+    var surname = document.getElementById("surname-grp");
+    var phoneNumber = document.getElementById("phoneNumber-grp");
+    var group = document.getElementById("name-surname-phone");
```

Comment:
Looks like some of the variables above are no longer needed - please delete unused variables.